rasa/importers/importer.py

```diff
@@ -21,7 +21,7 @@ class TrainingFileImporter:
         """Retrieves the domain of the bot.
 
         Returns:
-            Loaded `Domain`.
+            Loaded ``Domain``.
```

Comment:
@erohmensing I'm pretty sure that it's only single ticks in pydoc (that's not rich text)

Reply:
the way it renders on the docs page needs the double tics -- here is double
<img width="394" alt="Screenshot 2019-07-19 at 14 54 44" src="https://user-images.githubusercontent.com/22333512/61536590-42126080-aa35-11e9-990e-011dd336ecb2.png">
vs single 
<img width="334" alt="Screenshot 2019-07-19 at 14 55 16" src="https://user-images.githubusercontent.com/22333512/61536594-450d5100-aa35-11e9-9f64-e032f9306429.png">
with `make livedocs`

Reply:
Ah ok 🤔 Thing the plugin should handle this instead of changing things in our code.

Reply:
Can look into that in the docs sprint 😀

Reply:
In general i'm pretty unhappy with how our autoclass and automethod look tbh